crates/bevy_ui/src/lib.rs

```diff
@@ -110,6 +111,23 @@ impl Plugin for UiPlugin {
                 CoreStage::PostUpdate,
                 widget::image_node_system.before(UiSystem::Flex),
             )
+            .add_system_to_stage(
+                CoreStage::Update,
+                widget::update_slider_value.label(widget::UpdateSliderValue),
+            )
```

Comment:
Why is the `update_slider_value` added to both update and post update? Is that necessary?

Reply:
Yeah, I have no idea why I did that.